packages/server/src/fhir/repo.ts

```diff
@@ -2288,7 +2301,7 @@ export function getLookupTable(resourceType: string, searchParam: SearchParamete
   return undefined;
 }
 
-const REDIS_CACHE_EX_SECONDS = 24 * 60 * 60; // 24 hours in seconds
+const REDIS_CACHE_EX_SECONDS = 5 * 60; // 5 minutes in seconds
```

Comment:
This seems like a pretty impactful global change. Did you mean to do that in this PR?

Reply:
Ah, nope — thanks for the catch!